content/docs/conditional-rendering.md

```diff
@@ -42,15 +43,15 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**Experimente no CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Este exemplo renderiza um "greeting" diferente dependendo do valor da `isLoggedIn` prop.
 
-### Element Variables {#element-variables}
+### Variáveis de Elementos {#variáveis-de-elementos}
```

Comment:
It was agreed not to translate internal links.

```suggestion
### Variáveis de Elementos {#element-variables}
```

Reply:
Thanks for pointing that out. I was confused about translating the links.